### DISCUSSION TITLE
Welcome to the Tastyworks unofficial API Discussions!

### DISCUSSION THREAD

**Discussion by boyan-soubachov** [boyan-soubachov/tastyworks_api#58] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bandwiches** [comment]
THIS IS EXCITING!

**Comment by ScientiaCapital** [comment]
This is awesome guys, nice job and thank you for keeping this going.  This
is a killer platform to make transactions with exotic options that most
brokers don't allow, don't know how to technically allow it, etc.

**Comment by perigvennetier** [comment]
Awesome!!

Get ready for all my noob questions! ;)

**Comment by barrypearce** [comment]
Hi, this is a very useful API. How do I extract the latest bid and ask for an option in an option chain?

**Reply by BrettBiba** [reply]
I am trying to figure out the same thing. 🙂

**Reply by bandwiches** [reply]
Sorry all, time is limited right now.  I'll look into this and see what I can help out with.

**Reply by bandwiches** [reply]
It might be somewhere else but it's not part of the `option` or `option-chain` response from the API.  I might look around and see what I can find on my next update.

**Reply by barrypearce** [reply]
Yeah it's not in option chain. A sort of ugly work around would be to take the option data (ticker, strike, expiry) from the option chain then use the dxfeed streamer "Quote" setting to subscribe and grab a snapshot of the latest option prices for each option in the chain. There might be a better way to do this.

**Reply by gneely74** [reply]
Did you ever figure out how to do this with dxfeed?  Do you have an example?

**Reply by adamsherman** [reply]
Here ya go - not pretty, but I put an example here.  https://github.com/boyan-soubachov/tastyworks_api/issues/53

Afaik, only way to get after looking at all their endpoints is via the streaming dxfeed as that's where their market data comes from and they don't expose snapshots via api.tastyworks.com as they do for /order, /account etc...
